server/history/util.py

```diff
@@ -20,8 +21,10 @@
 
 
 def get_history_db_connection(with_db_name=True):
+    db_host = os.getenv("POSTGRES_HOST", "localhost")
+
     dbname = secrets.POSTGRES_DB if with_db_name else None
     if secrets.POSTGRES_PASS == '':
-        return psycopg2.connect(user=secrets.POSTGRES_USER, dbname=dbname)
+        return psycopg2.connect(user=secrets.POSTGRES_USER, dbname=dbname, host=db_host)
     else:
-        return psycopg2.connect(user=secrets.POSTGRES_USER, password=secrets.POSTGRES_PASS, dbname=dbname)
+        return psycopg2.connect(user=secrets.POSTGRES_USER, password=secrets.POSTGRES_PASS, dbname=dbname,  host=db_host)
```

Comment:
Remove extra space so the linter is happy

```suggestion
        return psycopg2.connect(user=secrets.POSTGRES_USER, password=secrets.POSTGRES_PASS, dbname=dbname, host=db_host)
```